python/spark_sklearn/test_utils.py

```diff
@@ -16,21 +16,32 @@
 from scipy.sparse import csr_matrix
 
 from pyspark import SparkContext
-from pyspark.sql import SQLContext
-from pyspark.mllib.linalg import Vectors
+from pyspark.sql import SparkSession
+from pyspark.ml.linalg import Vectors
 
-sc = None
+# Used as deocrator to have one PySpark SparkSession per fixture.
```

Comment:
More precisely, it is used to create and destroy a Spark session for each fixture